### ISSUE TITLE
Settings: At 200% resize mode controls present under 'Settings' blade are not visible while navigating over them.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1035]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1239268

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.
Go to Browser Settings->Zoom->200%.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate  to 'Settings' menu item and select it.
Navigate through all the controls under Settings blade.
Resize the web page up to 200% mode.
Navigate on controls at resize mode.
Verify the issue.

Actual Result:
At 200% resize mode controls present under 'Settings' blade are not visible while navigating over them.

Expected Result:
At 200% resize mode controls present under 'Settings' blade should be visible properly while navigating over them.

Note:
Similar issue exists at 400% mode.
This issue exists throughout the application for similar controls like Settings, New Keyspace, Add Table, Add, New Vertex blades for similar Data explorer page for different created Cosmos DB like 'Gremlin(graph)', 'Azure table', 'Cassandra' etc .

![image](https://user-images.githubusercontent.com/8519407/132719152-3bd63799-b1d6-4d1b-8c1c-2ae7c8a3ce0a.png)

![image](https://user-images.githubusercontent.com/8519407/132719166-ee5f4607-56b0-48ef-b226-24b7eba9b3ca.png)

**Comment by hardiknai-techm** [comment]
![image](https://user-images.githubusercontent.com/80053762/133260022-1041749b-effe-46f2-8aeb-47499ffcc360.png)

**Comment by ato9000** [comment]
AB#1239268
